docs/source/transforms.rst

```diff
@@ -261,6 +261,22 @@ The new transform can be used standalone or mixed-and-matched with existing tran
     AugMix
     v2.AugMix
 
+Cutmix - Mixup
```

Comment:
Nit: technically, paper names of these techniques are  CutMix and MixUp.

Reply:
Sorry, saw this after I merged. I'll address via https://github.com/pytorch/vision/issues/7766